x/programs/runtime/import_program.go

```diff
@@ -42,6 +42,19 @@ func extractErrorCode(err error) (ErrorCode, bool) {
 	return 0, false
 }
 
+func (e ErrorCode) Error() string {
+	switch e {
+	case CallPanicked:
+		return "The call panicked"
+	case ExecutionFailure:
+		return "An unexpected error happened"
```

Comment:
Shouldn't this be the catch-all instead of panicking? I don't think we want the runtime to panic like this.

Reply:
I just removed this `Error() string` function as it was supposed to be used because there was some back and forth conversion between the `ErrorCode` and `error` but it got solved.